CHANGELOG

```diff
@@ -1,6 +1,12 @@
 devel
 -----
 
+* COR-8: Improved performance of shard statistics on DBservers by updating only
+  metrics for databases with changes, rather than recalculating all metrics.
```

Comment:
This doesn't improve the performance: previously, already only the dirty databases were counted.

It does have a much more significant effect: previously the metrics did *not* account for *non*-dirty databases, so the metrics were, generally, completely off, unless in moments where all databases have been dirty during the last maintenance run.